### DISCUSSION TITLE
"Copy link to selection" is grayed out in context menu

### DISCUSSION THREAD

**Discussion by thistlillo** [RyotaUshio/obsidian-pdf-plus#240] [Q&A] [answered]
I have installed PDF++. It is the only community plugin installed.

When I select some words in the pdf and right click, I see this:
<img width="261" alt="Screenshot 2024-06-29 at 16 39 17" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/6717655/97241b6d-1103-4d16-b610-72c5c354d052">

The "copy link to selection" is disabled.

Using
MacOS Sonoma 14.5
Obsidian 1.6.5 (installer 1.6.3)
PDF++ 0.40.8 (enabled)

What am I supposed to check?

**Comment by puneetkhurana009** [comment]
I have the same issue. I am on Windows and The "copy link to selection" is disabled.. Was it solved for you ?

**Comment by RyotaUshio** [comment, marked as answer]
@thistlillo @puneetkhurana009 That "copy link to selection" part is meant to be a heading, not a disabled menu item, and you can click a menu item for each color to copy a link to selection with the selected color.

The reason why I put the heading in the first place is that, depending on your settings, the context menu can have another similar section like "Add highlight to file". Without the headings, you won't be able to tell what each menu item is for.

<img width="490" alt="image" src="https://github.com/user-attachments/assets/d4530f6e-0d17-48a9-90c5-3df8bc17db5a" />

That said, I do recognize this is not very straightforward UI... I'm planning to rewrite the context menu from scratch in the near future, but I'm not sure how I can make it clear that it's a heading and not a disabled menu item...
